ui/StatusQ/src/StatusQ/Controls/StatusDropdown.qml

```diff
@@ -108,7 +116,7 @@ QC.Popup {
             x: 0
             y: d.windowHeight - height
             width: d.windowWidth
-            height: Math.min(implicitHeight, d.windowHeight * 0.9)
+            height: root.fillHeightOnBottomSheet ? d.windowHeight * 0.85 : Math.min(implicitHeight, d.windowHeight * 0.9)
```

Comment:
make the second one 0.85 too ?

Reply:
or 85% 😆

Reply:
@caybro @Khushboo-dev-cpp I've solved the mixed values on `StatusDropdown` + added a constant instead with some comments to clarify why the value. Hope that's enough!